services/core-api/app/api/parties/response_models.py

```diff
@@ -135,7 +136,8 @@ def format(self, value):
         'now_party_appt': fields.Nested(NOW_PARTY_APPOINTMENT),
         'organization_guid': fields.String,
         'organization': fields.Nested(ORGANIZATION, skip_none=True),
-        "digital_wallet_connection_status": fields.String
+        "digital_wallet_connection_status": fields.String,
+        'middle_name': fields.String
```

Comment:
Is this right?  `middle_name` on the `PARTY_BUSINESS_ROLE_APPT` feels like a strange fit.

Reply:
This is the full screen of the code. It is actually on the Party reponse model which has first_name attribute. 
![image](https://github.com/bcgov/mds/assets/72251620/7aee830a-c992-4975-a37c-71bf3a100a09)

Reply:
Ah, okay!